aiohttp/protocol.py

```diff
@@ -255,6 +255,10 @@ def __call__(self, out, buf):
 
 class HttpPayloadParser:
 
+    CONTENT_LENGTH_ID = multidict.cistr('CONTENT-LENGTH')
+    SEC_WEBSOCKET_KEY1_ID = multidict.cistr('SEC-WEBSOCKET-KEY1')
+    TRANSFER_ENCODING = multidict.cistr('TRANSFER-ENCODING')
```

Comment:
Same thing. For ClientResponse you'd used TRANSFER_ENCODING_ID name, but here it's different. May be drop _ID thing at all?